generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-jms-artemis-client/pom.xml

```diff
@@ -57,7 +57,7 @@
     <dependency>
       <groupId>io.quarkiverse.artemis</groupId>
       <artifactId>quarkus-test-artemis</artifactId>
-      <version>3.3.0</version>
+      <version>${quarkus-qute-web.version}</version>
```

Comment:
Is that accurate?

Reply:
I have no idea but I've seen a similar issue before: https://github.com/quarkusio/quarkus-platform/pull/1234#discussion_r1665536264.